R/speed_aggregate.R

```diff
@@ -233,7 +233,7 @@ speed_aggregate <- function(x,rel,fname=NULL,weight=NULL,from=NULL,to=NULL,dim=1
     if(length(grep("\\.csv$",rel))>0) {
       rel <- getAggregationMatrix(rel,from=from,to=to) 
     } else {
-      rel <- read.spam(rel)
+      stop("cannot read spam files with this luscale version, please use luscale <= 2.27.12")
```

Comment:
```suggestion
      stop("cannot read spam files with this luscale version, please use luscale < 3.0.0")
```

Reply:
Funny, I had the exact same thing before, and then decided to switch it, so you have a concrete version that you can install directly. Do you still want to change it? I don't mind

Reply:
also not a strong preference. I thought it is more intuitive to clarify via the error message that the major version jump is also the cutoff point